### ISSUE TITLE
Vehicles turn around and accelerate to crazy speeds

### ISSUE THREAD

**Issue by zenna** [sisl/AutoViz.jl#30]
First of all, thanks for this package, and especially for putting in the work to make it well documented.

I've run into an issue where a car, for one reason or another, will decide to turn around and go the wrong way around the track.  From that point it accelerates wildly until its just zooming in and out of the screen every frame.

Without looking at the implementation it seems like it could be because there is no negative speed limit set?

I can construct a minimal example if this necessary but this happened on the first or second time I used the package, so I imagine you may have encountered it yourself.

Is there a solution?

**Comment by MaximeBouton** [comment]
Hi @zenna, I think a minimum working example would be useful to better corner the issue. 
Note that `AutoViz.jl` is only taking care of the visualization, the actual simulation happens in https://github.com/sisl/AutomotiveDrivingModels.jl


The behavior of the vehicle is controlled by whatever driver model and action type you specified in your simulation.

**Comment by zenna** [comment]
Yes. I know this.  I'm not sure why I posted the issue here.  I'll make the MWE
